ci-operator/step-registry/windows/e2e/operator/test/upgrade/windows-e2e-operator-test-upgrade-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: windows-e2e-operator-test-upgrade
+  steps:
+    - ref: windows-e2e-operator-test-upgrade-install
+    - ref: windows-e2e-operator-test-upgrade-setup
+    - ref: windows-e2e-operator-test-upgrade-upgrade
+    - ref: windows-e2e-operator-test-upgrade-test
+  documentation: |-
+    The windows-e2e-operator-test-upgrade upgrade step chain contains all the individual steps necessary to
+    end to end the upgrade scenario.
```

Comment:
`end to end the upgrade scenario` -> `test WMCO upgrades`

Reply:
Fixed.